packages/angular_devkit/build_optimizer/src/transforms/scrub-file.ts

```diff
@@ -6,8 +6,15 @@
  * found in the LICENSE file at https://angular.io/license
  */
 import * as ts from 'typescript';
+import { collectDeepNodes } from '../helpers/ast-utils';
 
 
+export const scrubFileRegexes = [
+  /decorators/,
```

Comment:
`indexOf()` is 10x faster than regexes. For static strings like those I'd use `indexOf`.  Reference: https://jsperf.com/regex-vs-strings